lib/blog-feed-loader.js

```diff
@@ -1,2 +1,6 @@
-// TODO: Replace this with a real implementation.
-module.exports = require('../test/browser/stub-blog-feed-loader');
+if (process.env.NODE_ENV !== 'production' &&
```

Comment:
Oh it doesn't have anything to do w/ the google feed API--it's `NODE_ENV` that the define plugin is using, so basically in production builds the above conditional will become

```javascript
if (false && require('./config').IN_TEST_SUITE) {
```

which will then get optimized out, since it's logically impossible for the conditional to ever be true. This means that `require('../test/browser/stub-blog-feed-loader')` will never be seen by webpack, so the stub blog feed loader won't be included in the JS bundle.

I just verified this on my local machine by running:

```
rm -rf dist
ORIGIN=blah NODE_ENV=production gulp
grep "Last week we announced" dist/*.js
```

The `grep` won't print anything. If, however, your remove the `process.env.NODE_ENV` from the conditional, running the same commands will print something, because it'll be including the stub loader (which contains the literal text "Last week we announced"), as the optimizer isn't certain that the conditional will always evaluate to `false`.